### DISCUSSION TITLE
Better touch for iOS

### DISCUSSION THREAD

**Discussion by abulka** [bcakmakoglu/vue-flow#259] [Ideas]
The drag drop example doesn't work on iOS Safari - I can't create new nodes by dragging them out.

I tried dragging out new edges but couldn't get that to happen either.

Suprisingly, most everything else worked OK on my iPad, so its close.

**Comment by bcakmakoglu** [comment]
Drag and Drop is not built into Vue Flow - you'll have to implement it yourself for mobile purposes.
Creating edges on mobile can be done by enabling `clickToConnect` and clicking one Handle to start and another to create an edge.
